tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md

```diff
@@ -122,8 +122,8 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUIFilesActionMenu/versions.feature:48](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesActionMenu/versions.feature#L48)
 -   [webUIFilesActionMenu/versions.feature:63](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIFilesActionMenu/versions.feature#L63)
 
-### [Change the file content of a received shared file](https://github.com/owncloud/ocis/issues/2319)
```

Comment:
Why are you overwriting the expected failure here instead of adding yours below?

Reply:
The `https://github.com/owncloud/ocis/issues/2319` issue was closed due to the inability to download the file into the "Shares" folder, and the test for this case has been changed (the `versions.feature:77` test now passes).

Reply:
Failed with line 77 🤔 https://drone.owncloud.com/owncloud/web/20990/17/22